packages/react-vapor/src/utils/UrlUtils.ts

```diff
@@ -0,0 +1,13 @@
+import * as QueryString from 'query-string';
+
+/* istanbul ignore next */
+function getSearchParams() {
+    const currentUrl = window.location.href;
+    const qPos = currentUrl.indexOf('?');
```

Comment:
I think this won't work if you have a real querystring and a querystring in the hash (e.g., `http://some.website.com/?dev#path/to/page?param=5`)

You might have to do a 
```suggestion
    const qPos = currentUrl.lastIndexOf('?');
```